packager/media/crypto/encryption_handler.cc

```diff
@@ -314,15 +317,15 @@ Status EncryptionHandler::ProcessMediaSample(
         total_size += subsample.clear_bytes;
       }
       if (subsample.cipher_bytes > 0) {
-        EncryptBytes(source, subsample.cipher_bytes, dest);
+        EncryptBytes(source, subsample.clear_bytes, dest, ciphertext_size);
```

Comment:
I think this should be cipher_bytes, not clear_bytes.  IIUC, the parameter is the number of input bytes to encrypt, which is cipher_bytes (to be encrypted), not clear_bytes (left clear).

Since it seems easy to make a mistake with that here, perhaps you should both revert that parameter and add a comment explaining.

Reply:
ah, ok, that makes sense, reverted and added a comment as you suggested.